### PR TITLE
Debian package is available

### PR DESCRIPTION
**Pull request by stephanlachnit**
Here: https://packages.debian.org/buster/gamemode
At the moment only for buster and sid.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ Issues with GameMode should be reported here in the issues section, and not repo
 ---
 ## Building and installing
 
-If your distribution already has GameMode packaged, it is preferable to install it directly from there. There are Solus, AUR, Gentoo and Fedora packages already available.
+If your distribution already has GameMode packaged, it is preferable to install it directly from there. There are Debian. Solus, AUR, Gentoo and Fedora packages already available.
```
Comment (aejsmith) [inline README.md]: Should be a comma after Debian.

**Comment by kakra** [comment]
Does this solve #70? Then maybe include ab Github close tag into the commit description.

**Comment by 9Strike** [comment]
> Does this solve #70? Then maybe include ab Github close tag into the commit description.
Sorry I don't know how to do that :/

**Comment by kakra** [comment]
@9Strike just write "Closes:" + the URL of that issue at the end of the message:

```
README: Add something

This is a multiline description
of what was fixed.

Closes: LINK-GOES-HERE
Signed-off-by: Your Full Name <your@email.domain>
```

Also, please squash your commits: `git rebase -i origin/master ` and then replace pick with squash in the lines that should be squashed with the previous commit. After squashing, git will open an editor to revise the commit message into its final form. This is when you can add the close and sign tags.

**Comment by stephanlachnit** [comment]
I think this can be closed now.

**Comment by aejsmith** [comment]
I added this to the README in 35af00895d270af86391e1f0da537925dbad4f49.

**Pull request closed without merging**